plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -308,6 +309,18 @@ public DeltaLakeConfig setCheckpointRowStatisticsWritingEnabled(boolean checkpoi
         return this;
     }
 
+    public boolean isCheckpointPartitionFilterEnabled()
+    {
+        return checkpointPartitionFilterEnabled;
+    }
+
+    @Config("delta.checkpoint-partition-filter.enabled")
```

Comment:
We may want to prune from the checkpoint file as well the column statistics which don't relate to the query.
Maybe a more generic name would be appropriate.

Suggestion: `delta.checkpoint-filtering.enabled`